spec/lib/constellation_spec.rb

```diff
@@ -22,5 +22,13 @@
         expect(e.message).to eq 'uh oh'
       end
     end
+    it 'raises an exception on a more generic http error' do
+      allow(Net::HTTP).to receive(:new).and_raise(SocketError.new('uh oh'))
```

Comment:
This was just a random place to hook into to raise one of the more generic errors.